src/pages/TxSender.vue

```diff
@@ -41,11 +41,28 @@
 					class="margin_top_med"
 					type="primary"
 					@click="openConfirmationModal"
-					:disabled="!txOutput"
+					:disabled="!txOutput || isSendDisabled"
 					>Send</a-button
 				>
 			</a-col>
 		</a-row>
+		<a-row v-if="confirmedResponse" class="margin_top_med">
+			<div>
+				<h1>Transaction Receipt</h1>
```

Comment:
h1 should be used for page title, here, for subsections we want probably h2 or h3